data_structures/src/superblock.rs

```diff
@@ -32,6 +35,8 @@ pub enum AddSuperBlockVote {
 pub struct SuperBlockState {
     // Set of ARS identities that will be able to send superblock votes in the next superblock epoch
     current_ars_identities: HashSet<PublicKeyHash>,
+    // current_ars_identities ordered by reputation
+    current_ordered_ars_identities: Vec<PublicKeyHash>,
```

Comment:
why is the reason to have two similar fields, one ordered and other not. It should be possible to keep only the ordered one?

Reply:
best performance of the hashset :P